zasm.ppj

```diff
@@ -0,0 +1,102 @@
+# 
+# PROJECT FILE generated by "Pelles C for Windows, version 8.00".
+# WARNING! DO NOT EDIT THIS FILE.
+# 
+
+POC_PROJECT_VERSION = 7.00#
+POC_PROJECT_TYPE = 3#
+POC_PROJECT_OUTPUTDIR = output#
+POC_PROJECT_RESULTDIR = .#
+POC_PROJECT_ARGUMENTS = #
+POC_PROJECT_WORKPATH = .#
+POC_PROJECT_EXECUTOR = #
+POC_PROJECT_ZIPEXTRA = #
+CC = pocc.exe#
+AS = poasm.exe#
+RC = porc.exe#
+LINK = polink.exe#
+SIGN = posign.exe#
+CCFLAGS = -std:C11 -Tx86-coff -Os -Ob1 -fp:precise -W1 -Gd -Go -DNO_DECLTYPE#
+ASFLAGS = -AIA32 -Gd#
+RCFLAGS = #
+LINKFLAGS = -subsystem:console -machine:x86 kernel32.lib advapi32.lib delayimp.lib#
+SIGNFLAGS = -location:CU -store:MY -timeurl:http://timestamp.verisign.com/scripts/timstamp.dll -errkill#
+INCLUDE = $(PellesCDir)\Include\Win;$(PellesCDir)\Include;..\include\asmdasm;include\asmdasm;src\asmdasm#
+LIB = $(PellesCDir)\Lib\Win;$(PellesCDir)\Lib#
+WizCreator = Pelle Orinius#
+
+# 
+# Build zasm.exe.
+# 
+zasm.exe: \
+	output\app.res \
```

Comment:
It's used to be "obj" dir for storing temporary object files. Let it be similar for all compilers?